code/__HELPERS/game.dm

```diff
@@ -328,15 +328,16 @@
 
 /proc/considered_alive(datum/mind/M, enforce_human = TRUE)
 	if(M && M.current)
-		if(M.has_antag_datum(/datum/antagonist/changeling))
-			return TRUE
+		var/is_cling = M.has_antag_datum(/datum/antagonist/changeling)
 		if(enforce_human)
 			var/mob/living/carbon/human/H
 			if(ishuman(M.current))
 				H = M.current
-			return M.current.stat != DEAD && !issilicon(M.current) && !isbrain(M.current) && (!H || H.dna.species.id != "memezombies")
+			return (!is_cling && M.current.stat != DEAD) && !issilicon(M.current) && !isbrain(M.current) && (!H || H.dna.species.id != "memezombies")

```

Comment:
This will make it so being a zombie nulls it even though you can easily get out zombification (might be your intention, idk), hence why I encourage you to expand this out to be less stupid.